02-express-tutorial/app.js

```diff
@@ -1 +1,69 @@
-console.log('Express Tutorial')
+const express = require('express');
+const { products } = require('./data')
+
+const app = express();
+
+app.use(express.static('./public')) 
+
+app.get('/api/v1/test', (req,res)=> {
+  app.get(res.json({message: 'It worked!'}))
+})
+
+app.get('/api/v1/products', (req,res)=> {
+  const newProducts = products.map((product) => {
+    const { id, name, image } = product
+    return { id, name, image }
+  })
+  res.json(newProducts)
+})
+
+app.get('/api/v1/productsall', (req,res)=> {
```

Comment:
Usually, the endpoint that returns all entities of a certain type is called simply the entity's name.  app.get('/api/v1/products');. It is implied that this will return all products (usually paginated).